packages/odata-common/src/entity.spec.ts

```diff
@@ -1,66 +1,44 @@
-import {
-  TestEntity,
-  TestEntityMultiLink,
-  TestEntitySingleLink
-} from '@sap-cloud-sdk/test-services/v2/test-service';
+import { CommonEntity } from '../test/common-entity';
 
 describe('entity', () => {
   describe('remote state', () => {
     it('setOrInitializeRemoteState() sets remote state on entity', () => {
-      const entity = TestEntity.builder()
-        .stringProperty('test')
-        .toSingleLink(
-          TestEntitySingleLink.builder().stringProperty('singleLink').build()
-        )
-        .toMultiLink([
-          TestEntityMultiLink.builder().stringProperty('multiLink').build()
-        ])
-        .withCustomFields({ custom: 'custom' })
-        .build()
-        .setOrInitializeRemoteState();
+      const entity = new CommonEntity();
+      entity.setOrInitializeRemoteState();
```

Comment:
Question: does it remove the tests for navigation properties?

Reply:
oh yes, because the links are v2 or v4 specific. I created a `entity.spec.ts` in v2 for that test